packages/drawer/rc/use-drawer.js

```diff
@@ -0,0 +1,81 @@
+import { useReducer } from 'react'
+
+import {
+  combineFns,
+  createActionCreator,
+  persistSynthetic,
+  pick,
+  removeUndefinedValues,
+  switchcaseF
+} from './utils.js'
+
+export const actionTypes = {
+  close: 'CLOSE',
+  open: 'OPEN',
+  reset: 'RESET'
```

Comment:
reset can be used when the component is controlled and receives new state from props